core/tests/snapshots.rs

```diff
@@ -218,6 +218,7 @@ mod tests {
             if slot % set_root_interval == 0 || slot == last_slot - 1 {
                 // set_root should send a snapshot request
                 bank_forks.set_root(bank.slot(), &request_sender, None);
+                bank.update_accounts_hash();
```

Comment:
@sakridge , this is required since we are no longer setting the hash on the bank. We used to do it [here](https://github.com/solana-labs/solana/blob/cc8be7869be5c352b381a3a75b4274d93a951841/runtime/src/accounts_background_service.rs#L91).
We get a default hash (111...) later [here](https://github.com/solana-labs/solana/blob/cc8be7869be5c352b381a3a75b4274d93a951841/core/tests/snapshots.rs#L157), if we don't set the hash in the bank sometime.
I'm not confident in the right way to fix these tests.

Reply:
we could calculate not using the index at some point. That would perhaps be more like the run-time behavior going forward. Perhaps it does not matter for these tests and this is sufficient.